File: library_common_business/src/main/java/cn/jack/library_common_business/loadsir/callback/LoadingCallback.java
package cn.jack.library_common_business.loadsir.callback;

import android.content.Context;
import android.view.View;
import com.kingja.loadsir.callback.Callback;
import cn.jack.library_common_business.R;


/**
 * Description:TODO
 * Create Time:2017/9/4 10:22
 * Author:KingJA
 * Email:kingjavip@gmail.com
 */

public class LoadingCallback extends Callback {

    @Override
    protected int onCreateView() {
        return R.layout.layout_loading;
    }


    @Override
    protected boolean onReloadEvent(Context context, View view) {
        return true;
    }
}

File: lib_base/src/main/java/com/jack/lib_base/uistate/loadsir/callback/LoadingCallback.kt
package com.jack.lib_base.uistate.loadsir.callback

import android.content.Context
import android.view.View
import com.jack.lib_base.R
import com.kingja.loadsir.callback.Callback

class LoadingCallback : Callback() {
    override fun onCreateView(): Int {
        return R.layout.layout_loading
    }

    override fun onReloadEvent(context: Context, view: View): Boolean {
        return true
    }
}